pkg/policy/l4.go

```diff
@@ -33,6 +33,8 @@ type L4Filter struct {
 	Port int
 	// Protocol is the L4 protocol to allow or NONE
 	Protocol string
+	// From Endpoints
+	FromEndpoints []api.EndpointSelector
```

Comment:
When the users print `cilium endpoint get XXX` they will see this in the output. It might be confusing:
```
"policy": {
    "allowed-consumers": [
      1,
      268,
      269
    ],
    "build": 60,
    "id": 269,
    "l4": {
      "egress": [],
      "ingress": [
        "{\n  \"Port\": 9080,\n  \"Protocol\": \"TCP\",\n \"FromEndpoints\":\"There will be duplicated and lots of verbose that here\"  \"L7Parser\": \"\",\n  \"L7RedirectPort\": 0,\n  \"L7Rules\": null,\n  \"Ingress\": true\n}"
      ]
    }
  },
```

I suggest you append  `json:"-"` at the end to omit this field from the output